relay-server/src/services/store.rs

```diff
@@ -896,13 +902,52 @@ impl StoreService {
                     false
                 }
             });
-        }
 
-        span.duration_ms = ((span.end_timestamp - span.start_timestamp) * 1e3) as u32;
-        span.event_id = event_id;
-        span.project_id = scoping.project_id.value();
-        span.retention_days = retention_days;
-        span.start_timestamp_ms = (span.start_timestamp * 1e3) as u64;
+            let group = span
+                .sentry_tags
+                .as_ref()
+                .and_then(|sentry_tags| sentry_tags.get("group").cloned())
+                .unwrap_or_default();
+
+            for (mri, summaries) in metrics_summary {
+                let Some(summaries) = summaries else {
+                    continue;
+                };
+                for summary in summaries {
+                    let Some(summary) = summary else {
```

Comment:
nit: You could [destructure](https://doc.rust-lang.org/rust-by-example/flow_control/match/destructuring/destructure_structures.html) `summary` here.